api/v3/ContributionRecurLineItem.php

```diff
@@ -24,6 +37,19 @@ function civicrm_api3_contribution_recur_line_item_create($params) {
   return _civicrm_api3_basic_create(_civicrm_api3_get_BAO(__FUNCTION__), $params);
 }
 
+/**
+ * ContributionRecurLineItem.createsubscriptionline API
+ *
+ * @param array $params
+ * @return array API result descriptor
+ * @throws API_Exception
+ */
+function civicrm_api3_contribution_recur_line_item_createsubscriptionline($params) {
```

Comment:
I think it's better to name the action in plural, as this is to create all lines that would apply for a payment plan. eg, `createsubscriptionlines`.